web/cltvspj/urls.py

```diff
@@ -1,8 +1,13 @@
-from django.urls import path
+from django.urls import path, include
 from . import views
 
+from rest_framework import routers
+from .api import CalculatorViewSet
+
+api_router = routers.DefaultRouter()
+api_router.register("calculator", CalculatorViewSet)
 
 urlpatterns = [
     path("", views.index, name="index"),
-    path("contact", views.contact, name="contact"),
```

Comment:
you need to add this line back that was removed

Reply:
ready!